src/statmodels.jl

```diff
@@ -79,20 +108,48 @@ when applicable the intercept and the distribution's dispersion parameter.
 """
 dof(obj::StatisticalModel) = error("dof is not defined for $(typeof(obj)).")
 
+"""
+    ess(obj::StatisticalModel)
+
+Return the the explained sum of squares.
+"""
+ess(obj::StatisticalModel) = sum(abs2, fitted(obj) - meanresponse(obj))
```

Comment:
Better use a generator to avoid allocating a copy. Same below.

Reply:
ess also means "effective sample size" in MCMC

Reply:
@nalimilan, could you give me a quick example to adopt it to using generators in this case?

Reply:
@simonbyrne Do you have a recommendation for a name with less ambiguity just in case?

Reply:
Something like `m = meanresponse(obj); sum(abs2(x - m) for x in fitted(obj))`.

Maybe we could use `mss` for "[Model sum of squares](https://en.wikipedia.org/wiki/Explained_sum_of_squares)"?